packages/sx.js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snapshot-labs/sx",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Shouldn't this be `0.1.1`?

Reply:
It's the default way changeset versions `0.1` (which I don't think is best myself), but it works OOB.

We can make it work like this instead:
breaking changes -> bump to 0.2
features or fixes -> bump to 0.1.1

but we will need to make those changes ourselves (manually tag breaking changes as `minor` and features as `patch` until we are out of `0.1`.

Reply:
@Sekhmet Got it thanks, if you can, please release SX.js as `0.1.1`, if too much hassle releave with `0.2.0` I want share the latest version with Yuki asap